### ISSUE TITLE
Cannot use nested Repeaters

### ISSUE THREAD

**Issue by BobbyBorisov** [laravel/nova-issues#5785]
- Laravel Version: 10.17.1
- Nova Version: 4.26.5
- PHP Version: 8.1.12

### Description:
I am trying to use nested Repeaters but it does not work.

### Detailed steps to reproduce the issue on a fresh Nova installation:

Create Eloquent Model and Nova Resource for Resource1
Create Eloquent Model, Nova Resource and Repeater for Resource2
Create Eloquent Model, Nova Resource and Repeater for Resource3

in Nova\Resource1 use Resource2Repeater
in Resource2Repeater use Resource3Repeater

Basically what I am trying to achieve is nested repeaters, but it is not working. 

It gives error `Attempt to read property \"connectors\" on array` in `HasMany:79`

Thanks!

**Comment by davidhemphill** [comment]
Duplicate of #5753

**Comment by BobbyBorisov** [comment]
@davidhemphill thanks for the quick answer! Do you have plans to support nested Repeaters in the future or it won't be possible?
